templates/base.html

```diff
@@ -18,12 +18,17 @@
     {% endcompress %}
     {% endblock foundation_css %}
 
+    {# Fontawesome #}
+    {% compress css %}
+    <link rel="stylesheet" href='{% static "external/css/font-awesome.min.css" %}'/>
+    {% endcompress %}
```

Comment:
Not sure whether it's worth it to compress font-awesome – the version you include is already minified, and it's big enough to be worth caching on its own.